### DISCUSSION TITLE
404 when trying to assign a domain to a project

### DISCUSSION THREAD

**Discussion by AopsConseil** [vercel/community#6138] [Bug Report]
### Page to Investigate

https://vercel.com/[user]/[project]/settings/domains

### Steps to Reproduce

When trying to assign a domain to a project I got a 404 
![Screenshot from 2024-03-15 16-27-11](https://github.com/vercel/community/assets/151153939/c18cd418-8c89-4faf-aa81-25b8b4795284)

**Comment by swarnava** [comment]
Hi,

Can you record a screencast and share?
